heroic-core/src/main/java/com/spotify/heroic/consumer/schemas/spotify100/v2/Value.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.consumer.schemas.spotify100.v2;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.auto.value.AutoValue;
+import com.google.protobuf.ByteString;
+
+/**
+ * Distribution histogram point value. Currently we
+ * support {@link Value.DoubleValue} and {@link Value.DistributionValue}
+ */
+@JsonSerialize(using = ValueSerializer.class)
+@JsonDeserialize(using = ValueDeserializer.class)
+public  abstract class Value {
```

Comment:
please leave a class comment - e.g. what it's for and where it's used.

Reply:
Please kindly ignore, I will probably remove this eventually. The way the code is structure at the moment metric is defined at every stage. I think  we will eventually use the definition in semantic-metric it has all the  documentation.

Reply:
Can you put that as a comment then? It's helpful knowledge.

Reply:
That's useful info, please put in the comment

Reply:
Can we please not dwell on the small stuffs. This is a  WIP in several repos with several PRs. We need to get the ball moving. As I said I will be replacing this file.

Reply:
I apologise for the accidental double-posts in several threads like this one. My browser crashed and didn't display the first set of posts so I did them again. I can see how they could be perceived as being irritating.

AFAIK, if the PR author doesn't agree with a comment, that's totally fine. They should simply leave an explanatory comment (like [the above](https://github.com/spotify/heroic/pull/700#discussion_r502425289_) and click "Resolve conversation".

Reply:
Apology accepted.